gpiozero/boards.py

```diff
@@ -2045,11 +2045,13 @@ class Energenie(SourceMixin, Device):
         Which socket this instance should control. This is an integer number
         between 1 and 4.
 
-    :param bool initial_value:
+    :type initial_value: bool or None
+    :param initial_value:
         The initial state of the socket. As Energenie sockets provide no
-        means of reading their state, you must provide an initial state for
+        means of reading their state, you may provide an initial state for
         the socket, which will be set upon construction. This defaults to
         :data:`False` which will switch the socket off.
+        Specifying :data:`None` will not switch the socket on construction.
```

Comment:
"will not alter the socket's state" might be less ambiguous than "will not switch the socket on construction" ?

("not switch the socket on" could be misinterpreted as "switch the socket off" ? :wink: )